app/kafka/manager/KafkaStateActor.scala

```diff
@@ -218,7 +383,16 @@ class KafkaStateActor(curator: CuratorFramework,
         sender ! getTopicDescription(topic)
 
       case KSGetTopicDescriptions(topics) =>
-        sender ! TopicDescriptions(topics.toIndexedSeq.map(getTopicDescription).flatten, topicsTreeCacheLastUpdateMillis)
+        sender ! TopicDescriptions(topics.toIndexedSeq.flatMap(getTopicDescription), topicsTreeCacheLastUpdateMillis)
+
+      case KSGetConsumerDescription(consumer) =>
+        sender ! getConsumerDescription(consumer)
+
+      case KSGetConsumedTopicDescription(consumer, topic) =>
+        sender ! getConsumedTopicDescription(consumer,topic)
+
+      case KSGetConsumerDescriptions(consumers) =>
```

Comment:
Which one? KSGetConsumerDescription gets used in the ClusterManagerActor, KSGetConsumers gets used in the KafkaManager, KSGetAllConsumerDescriptions gets used by the BrokerViewCacheActor. I guess KSGetConsumerDescriptions doesn't get used, you're right, I added it automatically, when I was thinking of having a list of valid consumer groups that the user could curate, and forgot to get rid of it.